ui/page/start_page.go

```diff
@@ -450,8 +454,16 @@ func (sp *startPage) loadingSection(gtx C) D {
 				if sp.loading {
 					return D{}
 				}
-				inset := layout.Inset{Top: values.MarginPadding100}
-				if sp.IsMobileView() {
+
+				var topMargin unit.Dp
+				if isSmallScreen {
+					topMargin = values.MarginPadding10
+				} else {
+					topMargin = values.MarginPadding100
+				}
+				inset := layout.Inset{Top: topMargin}
```

Comment:
```suggestion
				topMargin := values.MarginPadding100
				if isSmallScreen {
					topMargin = values.MarginPadding10
				}
				inset := layout.Inset{Top: topMargin}
```

Try to avoid if else statements where possible as it makes the code messy. the above is shorter and easier to read